Automattic/wp-calypso
# Modularise sites plans selectors

This PR is part of a larger effort that tries to remove plan data from the `entry-main` chunk. By splitting this monolithic module into individual ones, we can avoid pulling in plan data when using selectors that don't need it.

This PR likely won't produce any savings on its own, but rather only in combination with other changes.

#### Changes proposed in this Pull Request

* Modularise `state/sites/plans/selectors`

#### Testing instructions

There are no code changes (only moving things around), so it should be sufficient to ensure that the unit tests work correctly, and there are no new build errors or warnings.
